setup/ide.md

```diff
@@ -0,0 +1,64 @@
+# Setting up your editor (IDE)
+
+Integrated Development Environment for short IDE is much more than just text editor.
+These days it will come with all you need for your work
+
+* text editor
+* debugger
+* console
+* plugins
+
+When you are just starting to program you need only a simple text editor.
+However the more you want to do the the more tools you will need and here an IDE comes handy.
+
+These days one of the most popular one is VSCode and all examples in this gists will use it.
```

Comment:
These days one of the most popular ones is VSCode and all the examples in this gists will use it.